custom_theme/partials/footer.html

```diff
@@ -64,11 +64,6 @@
   </div>
 </footer>
 
-<!-- Start of HubSpot Embed Code -->
-<script type="text/javascript" id="hs-script-loader" async defer src="//js.hs-scripts.com/4795067.js"></script>
```

Comment:
Do we need to remove the actual JS as well?

Reply:
Oh never mind, I guess this is hosted elsewhere. I'm not familiar with this protocol-less url form here. How does that work? Does it just use the current protocol or something?

Reply:
Yea it's hosted externally by `hs-scripts.com`. I believe not specifying the scheme means use whatever scheme the current page is loaded in (which makes this a bad idea because it might not be `https?` at all!)